atat_internal_api.yaml

```diff
@@ -1912,13 +1918,43 @@ components:
       required:
         - name
       type: object
-      allOf:
-        - $ref: "#/components/schemas/BaseObject"
-        - $ref: '#/components/schemas/AppEnvAccess'
       properties:
         name:
           pattern: "^[a-zA-Z\\d _-]{1,100}$"
           type: string
+      # Remove the addition of 'allOf' while still using 'additionalProperties'
+      # which adds duplication to the spec and is not ideal, but allows for
+      # validation to occur for the middy validator
```

Comment:
This is an ajv specific issue: https://ajv.js.org/faq.html#additional-properties-inside-compound-keywords-anyof-oneof-etc @jeffsegal-ccpo @jasonburkert-ccpo

Reply:
Tried to resolve this using various open api patterns, but still receiving errors.

Reply:
Thanks for pointing to the ajv link for the source. Here is another link (you pointed me to previously) that is kind of related as well. 
- [Schema Composition with allOf](https://json-schema.org/understanding-json-schema/reference/combining.html#properties-of-schema-composition)

Reply:
If we want the increased security of `additionalProperties: false`, then compound keyword `allOf` must be removed.  This makes us repeat ourselves in the API spec, but will allow validation via [middy validator](https://middy.js.org/packages/validator/) which has a dependency on ajv.